app/api/tax.py

```diff
@@ -43,9 +65,38 @@ class Meta:
                          type_='event')
 
 
-class TaxList(ResourceList):
+class TaxListPost(ResourceList):
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
+            raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
+
+    def before_get(self, args, kwargs):
+        if has_access('is_admin'):
+            self.schema = TaxSchema
+        else:
+            self.schema = TaxSchemaPublic
+
+    schema = TaxSchema
```

Comment:
Public schema doesnt require logged in user, while coorganizer access is already being checked in before post. While only the admin can see all the tax with private details in a list.